### ISSUE TITLE
Nfc not work on latest a11 source stable and open beta 4

### ISSUE THREAD

**Issue by EviraKernel** [OnePlusOSS/android_kernel_oneplus_sm7250#4]
i compile latest open beta 4 and latest stable source a11 nfc not work on kernel

**Comment by SamMartello** [comment]
I can confirm the source code is still bugged, this issue should really be taken into consideration…

**Comment by KrossKinetic** [comment]
Yes , This still an issue and would request OnePlus to fix it as soon as possible
